atomix/cluster/src/test/java/io/atomix/raft/RaftFailOverTest.java

```diff
@@ -118,19 +117,97 @@ public void shouldRecoverLeaderRestart() throws Throwable {
   }
 
   @Test
-  @Ignore("https://github.com/zeebe-io/zeebe/issues/4467")
-  public void testNodeCatchUpAfterCompaction() throws Exception {
+  public void shouldDoSnapshot() throws Exception {
```

Comment:
Nit: snapshot is a verb, so just say `shouldSnapshot` ;)

Reply:
hm, but normally we say 'take a snapshot' right?